packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -854,11 +938,14 @@ class StripeHelper {
       }
 
       const product = await this.expandResource(
+        // @ts-ignore
```

Comment:
These are being ignored here, because this entire method is going to be eliminated and we're going to return raw objects to the client.